net_printer/README.md

```diff
@@ -0,0 +1,116 @@
+# FPGA Interchange Format Physical Netlist Net Printer `np.py`
+
+`np.py` is a simple python program for printing out nets from an FPGAIF `*.phys`
+file. This program is provided to aid in the development of a contestant router,
+by providing some visibility into the input and output files that contestant
+routers consume and emit.
+
+Consider the following example output:
+
+```
+$ python3 np.py ../vtr_mcml_rwroute.phys u_calc/boundaryChecker/r_ux__57_reg[21]_srl27___u_calc_dropSpin_photon29_o_sleftz_reg_r_n_0
+============================================================
+Route tree for net: u_calc/boundaryChecker/r_ux__57_reg[21]_srl27___u_calc_dropSpin_photon29_o_sleftz_reg_r_n_0
+
+    Source: 0
+    [{      belPin  SLICE_X84Y94 C6LUT O6
+     {      sitePIP SLICE_X84Y94 OUTMUXC D6 False
+            belPin  SLICE_X84Y94 OUTMUXC OUT
+         }  sitePin SLICE_X84Y94 CMUX
+     {   }  belPin  SLICE_X84Y94 OUTMUXC D6
+            sitePin SLICE_X84Y94 C_O
+     {   }  belPin  SLICE_X84Y94 C_O C_O
```

Comment:
This doesn't look correct. The latest benchmarks have this:
```
    [{      belPin  SLICE_X84Y94 C6LUT O6
     {      belPin  SLICE_X84Y94 OUTMUXC D6
            sitePIP SLICE_X84Y94 OUTMUXC D6 False
            belPin  SLICE_X84Y94 OUTMUXC OUT
            belPin  SLICE_X84Y94 CMUX CMUX
         }  sitePin SLICE_X84Y94 CMUX
            belPin  SLICE_X84Y94 C_O C_O
            sitePin SLICE_X84Y94 C_O
```

Reply:
Don't forget to explanation text below too.

Reply:
please see: 21375b89bb708c37ea993de441617d81958e7896